contrib_bots/lib/spotify.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from future import standard_library
```

Comment:
Yeah, because it's not in Zulip's reqs at the moment, so many developers won't have it installed. In order to make sure they know it's an additional req, a comment won't hurt. 

You probably had it already installed in your environment.